xskillscore/tests/test_probabilistic.py

```diff
@@ -570,7 +570,7 @@ def test_roc_auc_score_out_of_range_forecast(
         dim="time",
         return_results="area",
     )
-    assert float(area) == 0.0
+    assert float(area) == [.0, .5]
```

Comment:
that was the thing I mentioned in #236. dont understand why tests returns something one value sometimes the other

Reply:
Yeah strange. Want to add a comment in the code saying it should assert to 0 but during some CI runs it is 0.5. That way we understand it in the future.